src/daipedemo/_config/config_dev.yaml

```diff
@@ -3,7 +3,7 @@ imports:
 
 parameters:
   databricks:
-    address: '[Your workspace address]'
+    address: 'https://adb-8328309645514519.19.azuredatabricks.net/'
```

Comment:
We do not want to set Databricks Workspace URL here. DEMO project is not transformed into any template. Leave the placeholder as it was.

Reply:
The pipeline in daipe-project-ci-cd no longer accepts DBX_URL as an argument, so leaving it in secrets in Demo project is not an option.